.env.sample

```diff
@@ -1,28 +0,0 @@
-# Copy me to .env
```

Comment:
Can you restore this file? This is needed for the docker.

Reply:
Sure, done.